docs/core/get-started/sdk.md

```diff
@@ -16,24 +24,30 @@ The Java SDK is available via Maven, on Sonatype's [Central Repository](http://c
 </dependencies>
 ```
 
-You can also [download the JAR](../../java/chain-sdk-latest.jar) as a binary.
+**Gradle** users should add the following to `build.gradle`:
 
-## Ruby
+```
+compile 'com.chain:chain-sdk-java:1.0.1'
+```
 
-The Ruby SDK is available [via Rubygems](https://rubygems.org/gems/chain-sdk). Make sure to use the most recent version whose major and minor components (`major.minor.x`) match your version of Chain Core. Ruby 2.1 or greater is required.
+You can also [download the JAR](https://search.maven.org/remotecontent?filepath=com/chain/chain-sdk-java/1.0.1/chain-sdk-java-1.0.1.jar) as a binary.
```

Comment:
Do we need this? I'm not sure if Java developers think of Maven as a download host.